qualtran/bloqs/qrom_test.py

```diff
@@ -93,6 +100,57 @@ def test_qrom_1d(data, num_controls):
             )
 
 
+def test_qrom_1d_classical():
+    rs = np.random.RandomState()
+    data = rs.randint(0, 2**3, size=10)
+    sel_size = int(np.ceil(np.log2(10)))
+    qrom = QROM([data], (sel_size,), target_bitsizes=(3,))
+    for i in range(len(data)):
+        i_out, data_out = qrom.call_classically(selection=i, target0_=0)
+        assert i_out == i
+        assert data_out == data[i]
+
+
+def test_qrom_1d_classical_nonzero_target():
+    rs = np.random.RandomState()
+    data = rs.randint(0, 2**3, size=10)
+    sel_size = int(np.ceil(np.log2(10)))
+    qrom = QROM([data], (sel_size,), target_bitsizes=(3,))
+    for i in range(len(data)):
+        target_in = int('111', 2)
+        i_out, data_out = qrom.call_classically(selection=i, target0_=target_in)
```

Comment:
Can we also verify that `qrom.decompose_bloq().call_classically()` matches ?

Reply:
added!